pymc/tests/test_gp.py

```diff
@@ -767,21 +795,20 @@ def test_raises3(self):
                 B = pm.gp.cov.Coregion(1)
 
 
-@pytest.mark.xfail(reason="MvNormal was not yet refactored")
 class TestMarginalVsLatent:
     R"""
     Compare the logp of models Marginal, noise=0 and Latent.
     """
 
     def setup_method(self):
-        X = np.random.randn(50, 3)
-        y = np.random.randn(50) * 0.01
-        Xnew = np.random.randn(60, 3)
-        pnew = np.random.randn(60) * 0.01
+        X = np.random.randn(20, 3)
+        y = np.random.randn(20)
+        Xnew = np.random.randn(30, 3)
+        pnew = np.random.randn(30)
         with pm.Model() as model:
             cov_func = pm.gp.cov.ExpQuad(3, [0.1, 0.2, 0.3])
             mean_func = pm.gp.mean.Constant(0.5)
-            gp = pm.gp.Marginal(mean_func, cov_func)
+            gp = pm.gp.Marginal(mean_func=mean_func, cov_func=cov_func)
             f = gp.marginal_likelihood("f", X, y, noise=0.0, is_observed=False, observed=y)
```

Comment:
If `is_observed` is deprecated, should this be removed?

Reply:
yes, thank you for finding these!  took them out